www/source/partials/docs/_using-hab-service-groups.html.md.erb

```diff
@@ -12,7 +12,7 @@ In addition, multiple service groups can reside in the same ring. This allows da
 be shared with other members of the ring regardless of which group they are in.
 
 ## Joining a service group
-To join a service group, run your service either natively with the `hab start` command, or
+To join a service group, run your service either natively with the `hab sup run` command, or
```

Comment:
This (and the rest of issues in this file) has already been fixed (via a fairly major prose change). See https://www.habitat.sh/docs/using-habitat/